src/main/java/liquibase/ext/snowflake/sqlgenerator/SnowflakeDropDefaultValueGenerator.java

```diff
@@ -8,7 +8,7 @@
 import liquibase.sqlgenerator.core.DropDefaultValueGenerator;
 import liquibase.statement.core.DropDefaultValueStatement;
 
-public class DropDefaultValueSnapshotGenerator extends DropDefaultValueGenerator {
+public class SnowflakeDropDefaultValueGenerator extends DropDefaultValueGenerator {
```

Comment:
@KushnirykOleh -- is this standard with naming conventions in other snapshot classes? i.e. to drop the word `Snapshot` from the class name?